sdk/keyvault/azure-security-keyvault-administration/test/ut/backup_restore_client_base_test.hpp

```diff
@@ -84,7 +85,7 @@ namespace Azure {
       m_keyVaultHsmUrl = GetEnv("AZURE_KEYVAULT_HSM_URL");
 
       // Options and credential for the client
-      BackupClientOptions options;
+      Azure::Security::KeyVault::Administration::Models::BackupClientOptions options;
```

Comment:
As I mentioned in chat, typically ClientOptions aren't in the Models namespace, unless they are sent either to the service or received from the service.

They should probably be in the Administration namespace.